src/features/meetings/monthly_view/index.tsx

```
@@ -0,0 +1,1164 @@
+                      week={value}
+                      key={`lc-part1-${index}`}
+                      label={t('tr_conductor')}
+                      type={AssignmentCode.MM_LCPart}
+                      assignment="MM_LCPart1"
+                      readOnly={false}
+                    />
+                  </WeekHoverBox>
+                </>
+              ) : (
+                <Box flex={1} />
+              );
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)